File: Java/Estrutura repetitiva/senha.java
// Importando biblioteca
import java.util.Scanner;

public class senha {

	public static void main(String[] args) {
		// Variável
		int senha;
		
		// Para facilitar a leitura
		Scanner sc = new Scanner(System.in);
		
		// Entrada de dados
		System.out.print("Digite a senha: ");
		senha = sc.nextInt();
		
		// Estrutura repetitiva
		while (senha != 2024) {
			System.out.print("Senha inválida! Tente novamente: ");
			senha = sc.nextInt();
		}
		
		// Saída de dados
		System.out.print("Acesso permitido!");
		
		sc.close(); // Boa prática de programação
	}
}

File: C++/Estrutura repetitiva/senha.cpp
// Biblioteca(s)
#include <iostream>

using namespace std;

int main()
{
    // Variável
    int senha;

    // Entrada de dados
    cout << "Digite a senha: ";
    cin >> senha;

    // Estrutura repetitiva
    while (senha != 2024) {
        cout << "Senha invalida! Tente novamente: ";
        cin >> senha;
    }

    // Saída de dados
    cout << "Acesso permitido!";

    return 0;
}
